### PR TITLE
Garnett card cleaning 

### PR DESCRIPTION
**Pull request by blongden73**
## What does this change?
Changes the way icons display on media cards
Makes sure the headline is always aligned correctly
Fixes sublinks and standfirst on immersive cards
Fixes an issue with the colour of immersive cars when they are small (mobile media list and tablet list)

### CODE REVIEW

File: common/app/views/fragments/items/facia_cards/contentCard.scala.html [hunk 1]
```diff
@@ -62,14 +62,12 @@
         @if(item.contentType.name.toLowerCase == "audio") {
             <div class="fc-item__media-meta">
                 @inlineSvg("volume-high", "icon")
-                Podcast
```
Comment (NataliaLKB) [inline common/app/views/fragments/items/facia_cards/contentCard.scala.html]: Could we have something like this instead:

```
<span class="u-h">Podcast</span>
```

That way users who are navigating without site will still know that this means.

File: static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-media.scss [hunk 2]
```diff
@@ -25,34 +27,19 @@
...
-            }
-
-            &::after {
-                display: inline-block;
-                content: '';
-                height: $gs-baseline + 1px;
-                margin: 6px;
-                width: 2px;
+                width: 14px;
+                padding: 0px 5px;
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-media.scss]: `0px $gs-gutter/4` ?

**Review by NataliaLKB — APPROVED** [review]
Left two comments, but after those change this looks good to go!

File: common/app/views/fragments/items/facia_cards/contentCard.scala.html (hunk 1 above)
Comment (blongden73) [reply common/app/views/fragments/items/facia_cards/contentCard.scala.html]: @NataliaLKB I have taken the words out. It's now just the icon.

File: static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-media.scss (hunk 2 above)
Comment (blongden73) [reply static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-media.scss]: done.

**Comment by NataliaLKB** [comment]
Thanks @blongden73 for making this change!

**Pull request merged**